src/server_manager/ui_components/outline-manual-server-entry.html

```diff
@@ -373,8 +406,18 @@
       clear: function() {
         this.$.serverConfig.value = "";
         this.showConnection = false;
-        this.$.awsDropDown.hide();
-        this.$.gcpDropDown.hide();
+        for (const dropdown of Polymer.dom(this.root).querySelectorAll(".instructions-collapse")) {
+          dropdown.hide();
+        }
+      },
+      // TODO(cohenjon) Provide encapsulation for cloud-specific details once we overhaul the AWS directions
+      _cloudProviderChanged: function(newProvider) {
+        this.cloudProviderName = this._computeCloudProviderName(newProvider);
```

Comment:
It shouldn't be necessary to call `_compute*` methods; the computed properties get updated when `cloudProvider` changes. You can make `installScriptStepNumber` a computed property with bindings to `isGenericCloudProvider`. Same with `pasteJsonStepNumber` bound to `installScriptStepNumber`.

Reply:
Done in d782227